src/app/drive/components/ShareDialog/ShareDialog.tsx

```diff
@@ -175,31 +198,33 @@ const ShareDialog = (props: ShareDialogProps): JSX.Element => {
   }, [itemToShare, roles]);
 
   const loadShareInfo = async () => {
+    if (!itemToShare?.item) return;
+
+    const isItemNotSharedYet = !isAdvanchedShareItem(itemToShare?.item) && !itemToShare.item.sharings?.length;
+    if (isItemNotSharedYet) return;
+
     setIsLoading(true);
     // Change object type of itemToShare to AdvancedSharedItem
     let shareAccessMode: AccessMode = 'public';
     let sharingType = 'public';
+    let isAlreadyPasswordProtected = false;
 
-    if (props.isDriveItem) {
-      sharingType = (itemToShare?.item as DriveItemData & { sharings: { type: string; id: string }[] }).sharings?.[0]
```

Comment:
We dont really need this check. The` sharingType` returned by the backend in the drive section is not accurate anymore since we implemented invites on public sharings.

We should always use the getSharingType endpoint to get the sharing info.